[ADD] Referrals: new points doc (#10134)
odoo/documentation: content/applications/hr/referrals/points.rst

===============
Referral points
===============

The *Referrals* application allows employees to earn points that are then exchanged for
:doc:`rewards <rewards>`.

Current referral points can be viewed on the main *Referrals* app dashboard, which appears when the
application is opened, after the :ref:`onboarding <referrals/onboarding>` slides have been viewed or
skipped.

At the top, a summary of the user's current points is displayed. The left side of the summary

> displays the total points earned, and the right side displays the points that are available to
> spend.

Reviewer: Small change to use the guilabel text :)
```suggestion
displays the :guilabel:`Total` points earned, and the right side displays the points that are
available :guilabel:`To Spend`.
```

To view the various referral stages, and their respective earned points, click one of the referral

> status icons beneath the avatar. The options are: :guilabel:`Referrals`, :guilabel:`Ongoing`, and
> :guilabel:`Successful`.

Reviewer: These are not necessarily icons, so I would rather:
```suggestion
statuses beneath the avatar. The options are: :guilabel:`Referrals`, :guilabel:`Ongoing`, and
:guilabel:`Successful`.
```

The current number of referrals that are still active in the recruitment pipeline, but have not yet
been hired or refused, appear above :guilabel:`Ongoing`. The number of referrals that have been
hired, appear above :guilabel:`Successful`. The total number of referrals, both the ongoing and
successful referrals combined, appears above :guilabel:`Referrals`.

My referrals
============

To see all the referrals, both ongoing and successful, click :guilabel:`Referrals`. The

> :guilabel:`My Referral` screen page loads, and displays all the referrals, with each individual

Reviewer: Nitpick to streamline
```suggestion
:guilabel:`My Referral` screen page displays all the referrals, with each individual
```

referral housed in its own referral card.

A successful referral displays a white :icon:`fa-check` :guilabel:`Hired` badge in the top-right
corner of the card, along with a vertical green stripe on the left-side of the card. Referrals that
are in process have a purple :guilabel:`In Progress` badge in the top-right corner.

Each referral card lists the name of the applicant, the subject/application title as it appears on
the applicant card in the *Recruitment* app, the name of the job position, the person responsible
for filling the role (i.e. the recruiter), and the points earned.

For referrals that have been hired, the selected avatar also appears on the card.

.. image:: points/referrals.png
   :align: center
   :alt: Two referral cards, one hired and one in-progress.

Points
------

The points that can be earned for a referral are the same across all job positions. Each stage of
the recruitment process has corresponding points assigned to it. The stages listed correspond to the
stages configured in the *Recruitment* application.

> On each individual referral card, beneath the points earned, is a bar chart that displays how many
> points have been earned out of the total possible points that can be earned, if the applicant is
> hired.

Reviewer: The progress bar is a better term for this UI element IMO:
```suggestion
On each individual referral card, beneath the points earned, is a progress bar that displays how
many points have been earned out of the total possible points that can be earned, if the applicant
is hired.
```

===============
Referral points
===============

The *Referrals* application allows employees to earn points that are then exchanged for
:doc:`rewards <rewards>`.

Current referral points can be viewed on the main *Referrals* app dashboard, which appears when the

> application is opened. At the top, a summary of the user's current points is displayed. The left

Author: oof - yes, I incorrectly assumed the user opened this AFTER the onboarding - which only appears once, ever. I'll add that!

side of the summary displays the total points earned, and the right side displays the points that
are available to spend.

To view the various referral stages and their respective earned points, click one of the buttons
directly beneath the avatars. The options are :guilabel:`Referrals`, :guilabel:`Ongoing`, and
:guilabel:`Successful`.

The current number of referrals that are still active in the recruitment pipeline, but have not yet
been hired or refused, appear above the :guilabel:`Ongoing` button. The number of referrals that

> have been hired appears above the :guilabel:`Successful` button. The total number of referrals, both

Reviewer 2: ```suggestion
have been hired, appear above :guilabel:`Successful`. The total number of referrals, both
```

> have been hired appear above the :guilabel:`Successful` button. The total number of referrals (both
> the ongoing and successful referrals combined) appear above the :guilabel:`Referrals` button.

Reviewer 3: ```suggestion
have been hired appears above the :guilabel:`Successful` button. The total number of referrals, both
the ongoing and successful referrals combined, appears above the :guilabel:`Referrals` button.
```

> have been hired appears above the :guilabel:`Successful` button. The total number of referrals, both
> the ongoing and successful referrals combined, appears above the :guilabel:`Referrals` button.

Reviewer 2: ```suggestion
have been hired, appear above :guilabel:`Successful`. The total number of referrals (the ongoing and successful referrals combined), appear above :guilabel:`Referrals`.
```

Reviewer 2: They don't really seem like "traditional" *buttons*, in my opinion. What do you think?

Author: True- I didn't know what to call them since they have an outline and it looks like a button when you hover. But I can eliminate the word 'button'.

.. note::
   To view the referrals associated with each of the categories, :guilabel:`Referrals`,
   :guilabel:`Ongoing`, and :guilabel:`Successful`, click on the corresponding button. All the

>    referrals for that specific category are displayed.

Reviewer 2: this admonition seems a bit redundant...

Author: ...an obvious. Axed!

My referrals
============

> To see all the referrals (both ongoing and successful) click the :guilabel:`Referrals` button. The

Reviewer 3: ```suggestion
To see all the referrals, both ongoing and successful, click the :guilabel:`Referrals` button. The
```

> :guilabel:`My Referral` screen loads. This screen displays all the referrals, with each individual

Reviewer 3: ```suggestion
:guilabel:`My Referral` screen loads, and displays all the referrals, with each individual
```

> :guilabel:`My Referral` screen loads, and displays all the referrals, with each individual referral

Reviewer 2: ```suggestion
:guilabel:`My Referral` page loads, and displays all the referrals, with each individual referral
```

housed in its own referral card.

A successful referral displays a white :icon:`fa-check` :guilabel:`Hired` badge in the top-right

> corner of the card, along with a vertical green stripe along the left-side of the card. Referrals

Reviewer 2: ```suggestion
corner of the card, along with a vertical green stripe on the left-side of the card. Referrals
```

> that are in process display a purple :guilabel:`In Progress` badge in the top-right corner.

Reviewer 2: ```suggestion
that are in process have a purple :guilabel:`In Progress` badge in the top-right corner.
```

> Each referral card lists the name of the applicant, the subject/application title as it appears on
> the applicant card in the *Recruitment* app, the name of the job position, the person responsible
> for filling the role (the recruiter), and the points earned.

Reviewer 2: ```suggestion
Each referral card lists the name of the applicant, the subject/application title (as it appears on
the applicant card in the *Recruitment* app), the name of the job position, the person responsible
for filling the role (i.e. the recruiter), and the points earned.
```

For referrals that have been hired, the selected avatar also appears on the card.

.. image:: points/referrals.png
   :align: center
   :alt: Two referral cards, one hired and one in-progress.

Points
------

The points that can be earned for a referral are the same across all job positions. Each stage of
the recruitment process has corresponding points assigned to it. The stages listed correspond to the
stages configured in the *Recruitment* application.

> On each individual referral card, beneath the points earned is a bar chart that displays how many

Reviewer 2: ```suggestion
On each individual referral card, beneath the points earned, is a bar chart that displays how many
```

> points have been earned out of the total possible points that can be earned if the applicant is

Reviewer 2: ```suggestion
points have been earned out of the total possible points that can be earned, if the applicant is
```

hired.

> Beneath the bar chart is a list of the various recruitment stages and the points earned when the

Reviewer 2: ```suggestion
Beneath the bar chart is a list of the various recruitment stages, and the points earned when the
```

> referral moves to that stage. If a stage has been achieved and the points have been earned, a green

Reviewer 2: ```suggestion
referral moves to that stage. If a stage has been achieved, and the points have been earned, a green
```

:icon:`fa-check` :guilabel:`(checkmark)` appears next to the stage.

> .. example::

Reviewer 3: I don't think this fits as an example. I would consider changing it to a note.

   The preconfigured point structure for referrals is as follows:

   - :guilabel:`Initial Qualification`: 1 point
   - :guilabel:`First Interview`: 20 points
   - :guilabel:`Second Interview`: 9 points
   - :guilabel:`Contract Proposal`: 5 points

>    - :guilabel:`Contract Signed`: 50 points

Reviewer 2: ![Screenshot 2024-07-09 121329](https://github.com/odoo/documentation/assets/73958186/2e1fc418-c40c-4868-8f30-f69bf362fd48)

I am seeing different default figures in the runbot I am using for this review. During your revisions, please double-check the accuracy of these figures

Author: I did have to double check with a blank database when I originally revised/made this doc, because ALL the Runbots have made changes to the default point structure. So what is in the doc is accurate for when you freshly install Referrals. Why they changed it in Runbot (since I don't see the value in it) is beyond me =D

   The default total points earned for a referral that is hired is 85 points. Modifications of the

>    points earned for each stage are done in the *Recruitment* application. Refer to the

Reviewer 3: ```suggestion
   points earned for each stage are made in the *Recruitment* application. Refer to the
```
